data/migrations/old migrations/20210422144958_accuracycolumn.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.alterTable('twitter_incidents', (tbl) => {
```

Comment:
We're getting rid of twitter_incidents and adding confidence to the new table - might not need this file.

Reply:
this file is in the old migrations folder